FluidNC/src/Spindles/HBridgeSpindle.h

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2022 -	Santiago Palomino
+// Use of this source code is governed by a GPLv3 license that can be found in the LICENSE file.
+
+#pragma once
+
+/*
+	This is a full PWM spindle driver without speed compensation for an H Bridge controller
+   with support for two directions. The external HW has the following PINS:
+     enable_pin : optional.
+     output_a_pin : Clockwise PWM signal
+     output_b_pin : Counter Clockwise PWM signal
+     When the output A is toggling, B is set LOW, and viceversa.
+
+     Features which could be added afterwards:
+     # Soft start to prevent unnecessary current spikes on the spindle supply. The spindleDelay function seems to be a way to delay the application from using a spindle which has not accelerated yet. A similar feature could be added completing or replacing that functionality for this spindle type.
+
```

Comment:
Line too long.  Why the # at the beginning?  In fact, just eliminate the whole "Features which could" section because if we add soft start, it should be done more comprehensively instead of inside one driver.

Reply:
Simplifying the comment. Anyway, I think it is good to have a disclaimer as this is a limitation which is only apparent for this kind of driver. Most of other spindles types have their own power supply. So, their drivers do not need to consider soft starting. In this case, not doing soft start when using a lab power supply with current limit, makes the input voltage drop, which in my particular case triggers a safe shutdown.